torch/csrc/dynamo/compiled_autograd.h

```diff
@@ -222,6 +227,17 @@ class CompiledNodeArgs {
     }
   }
   template <typename T>
+  void collect(const std::unordered_set<T>& s) {
+    collect_size(s.size());
+
+    std::vector<T> v(s.begin(), s.end());
+    // stable sort over keys to ensure deterministic collect order
+    std::stable_sort(v.begin(), v.end());
```

Comment:
Not all types will be sortable.  Which variable is triggering this?  Is it even needed?

Reply:
https://github.com/pytorch/pytorch/blob/368f242e377108a29db9ae821126a9df4b6e40a0/torch/csrc/autograd/custom_function.h#L143-L144

Both are needed because they have public setters, and can be called dynamically in forward
https://github.com/pytorch/pytorch/blob/368f242e377108a29db9ae821126a9df4b6e40a0/torch/csrc/autograd/custom_function.h#L118-L125

Reply:
Are these actually used for the needed callbacks?

Reply:
what does "needed callbacks" refer to? not used in fbgemm

Reply:
If they aren't used in fbgemm, perhaps just assert they are empty then we won't need this code.